src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -308,6 +308,12 @@ void CascadiaSettings::_CreateDefaultKeybindings()
                               KeyChord{ KeyModifiers::Ctrl | KeyModifiers::Shift,
                                         static_cast<int>('9') });
 
+    keyBindings.SetKeyBinding(ShortcutAction::IncreaseZoom,
+                              KeyChord{ KeyModifiers::Ctrl,
+                                        VK_ADD | VK_OEM_PLUS });

```

Comment:
Doesn't `VK_ADD | VK_OEM_PLUS` evaluate to `0x6B | 0xBB` which is just `0xFB‬` - `VK_ZOOM`?

I'd avoid the `VK_OEM_*` keys in the defaults, since we can't serialize them ATM #1212